packages-nextgen/kestrel_core/src/kestrel/exceptions.py

```diff
@@ -0,0 +1,29 @@
+class KestrelError():
+    pass
+
+class InstructionNotFound(KestrelError):
+    pass
+
+class InvalidInstruction(KestrelError):
+    pass
+
+class InvalidSeralizedGraph(KestrelError):
+    pass
+
+class InvalidSeralizedInstruction(KestrelError):
+    pass
+
+class VariableNotExist(KestrelError):
```

Comment:
I know we have this exception already in Kestrel, but I find the name doesn't read well.  "Not exist" is odd phrasing; maybe `NonexistentVariable` or `VariableNotFound`?